matplotlib_demo.py

```diff
@@ -3,80 +3,91 @@
 import matplotlib.pyplot as plt
 from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
 from matplotlib.figure import Figure
-from tkinter import *
-import random
-import timeline
+from datetime import date
+
+
+class Stats():
+    def __init__(self):
+        self.dayPercent = []
+        self.moods = []
+        self.dates = []
+
+    def set_variables(self, t):
+        """
+        Will loop through the timeline object and append computed data
+        to its own dataList UNTIL IT REACH TODAY:
+        self.dayPercent = [50, 40, 30, 70, 70, 80, ...]
+        self.moods = [1, 2, 4, 1, 2, 2, 5, ...]
+        self.dates = [(2020, 4, 1), (2020, 4, 2), (2020, 4, 3), (2020, 4, 4), ...]
+        """
+        week = 1
+        day = 0
```

Comment:
```suggestion
        week = 1
        day = 0
```

Are we going to be given this info by panel?

Reply:
no these are just instances variables needed to loop through the timeline object